apps/climatemappedafrica/src/components/DropdownSearch/index.js

```diff
@@ -112,16 +112,34 @@ function DropdownSearch({
     }
   };
 
-  const icon =
-    !suggestions?.length || variant === "explore" ? iconProp : SearchIcon;
   const searchIconButton = (
     <IconButton
       color="primary"
       onClick={handleClickSearch}
       size="small"
       className={classes.button}
     >
-      <Image src={icon} width={48} height={48} alt="search" />
+      {variant === "explore" ? (
+        <SvgIcon
+          component={IconProp}
+          viewBox="0 0 48 48"
+          sx={{
+            width: 48,
+            height: 48,
+          }}
+        />
+      ) : (
+        <SvgIcon
+          component={SearchIcon}
+          viewBox="0 0 48 48"
+          sx={{
+            width: 48,
+            height: 48,
+            borderRadius: "50%",
+            border: "2px solid #fff",
+          }}
+        />
+      )}
```

Comment:
Without any duplicates:

```js
  let iconComponent = SearchIcon;
  let iconBorder = undefined;
  if (variant === explore) {
    iconComponent = IconProp;
    iconBorder = {
      borderRadius: "50%",
      border: "2px solid #fff",
    };
  }
```
Then
```suggestion
      <SvgIcon
        component={iconComponent}
        viewBox="0 0 48 48"
        sx={{
          width: 48,
          height: 48,
          ...iconBorder,
        }}
      />
```